server/src/main/java/io/druid/segment/indexing/RealtimeTuningConfig.java

```diff
@@ -55,7 +55,14 @@
 
   private static File createNewBasePersistDirectory()
   {
-    return Files.createTempDir();
+    try {
+      return Files.createTempDir();
+    }
+    catch (IllegalStateException e) {
+      String message = "Failed to create temporary directory!" + System.lineSeparator() +
+              "Make sure `java.io.tmpdir` property is set to existing and writable directory.";
```

Comment:
Grammar: "Make sure the `java.io.tmpdir` property is set to an existing and writable directory."

Reply:
fixed